docs/user/introduction.rst

```diff
@@ -37,7 +37,6 @@ The GeoMesa website may be found at http://www.geomesa.org/. For additional info
 * The GeoMesa FAQ: http://www.geomesa.org/faq/
```

Comment:
semi-related: should we remove this reference for now? the faqs are pretty out of date, no?